lib/routes/account.js

```diff
@@ -972,10 +972,6 @@ module.exports = function (
         log.begin('Account.RecoveryEmailResend', request)
         var sessionToken = request.auth.credentials
         var service = request.payload.service || request.query.service
-        if (sessionToken.emailVerified ||
```

Comment:
So we also want to skip short-circuiting if the email has been already been verified? Seems fine, I just want to confirm that its intentional.